PEAR.php

```diff
@@ -89,64 +89,64 @@ class PEAR
     /**
      * Whether to enable internal debug messages.
      *
-     * @var     bool
+     * @var bool
      * @access  private
      */
-    var $_debug = false;
+    public $_debug = false;
 
     /**
      * Default error mode for this object.
      *
-     * @var     int
+     * @var int
      * @access  private
      */
-    var $_default_error_mode = null;
+    public $_default_error_mode = null;
```

Comment:
Bother - the joys of having a preWrite hook in vim that I'd forgotten about... 
I'll have a look at this in the morning.